manifests/n/Novicon/Munixo/Client/5.0.65.14161/Novicon.Munixo.Client.yaml

```
@@ -0,0 +1,23 @@
+PackageIdentifier:            Novicon.Munixo.Client
+PackageVersion:               5.0.65.14161
+PackageLocale:                en-US
+Publisher:                    Novicon
```

```suggestion
Publisher: Novicon GmbH
```
This must match what is shown in control panel